.github/workflows/int-and-connected-test-run.yml

```diff
@@ -1,153 +1,85 @@
     steps:
-      - name: "Checkout code"
-        uses: actions/checkout@v3
+      - name: Checkout code
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: "Cache build"
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.CACHE_PATHS }}
-          key: ${{ runner.os }}-build-unit }}
-      - name: "Run unit tests and sonar scan via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
         with:
-          actions_subcommand: 'gradleinttest'
-          pgport: ${{ job.services.postgres.ports[5432] }}
-          test_to_run: 'check'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-          sonar_token: ${{ secrets.SONAR_TOKEN }}
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'gradle'
+      - name: Run unit tests
+        env:
+          GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          TDR_LOG_APPENDER: 'Console-Standard'
+        run: |
+          # extract service account credentials
+          base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
+          # assemble code, run unit tests, and generate scan
+          ./gradlew assemble
+          ./gradlew check --scan jacocoTestReport sonar
   test_connected:
-    name: "Run connected tests"
-    outputs:
-      job-status: ${{ job.status }}
+    name: Connected tests
+    runs-on: ubuntu-latest
     timeout-minutes: 180
-    needs: test_check
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    if: "!contains( github.event.pull_request.labels.*.name, 'skip-ci')"
-    runs-on: ${{ matrix.os }}
-    ## skips if pr label is 'skip-ci'
-    # run a local Postgres container in Docker for the basic check tests
-    services:
-      postgres:
-        image: postgres:11
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: postgres
-        ports:
-          - 5432:5432
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
     steps:
-      - name: "Checkout code"
-        uses: actions/checkout@v3
-      - name: "Cache build"
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.CACHE_PATHS }}
-          key: ${{ runner.os }}-build-connected
-      - name: "Write RBS SA to a file"
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'gradle'
+      - name: Run connected tests
+        env:
+          # required for azure tests
+          AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
+          AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
```

Comment:
Is that the case here that these are similar to google client id values that are not intended to be secret?

Reply:
Yes, that's correct.